glean-core/android/src/main/java/mozilla/telemetry/glean/rust/RustBuffer.kt

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.telemetry.glean.rust
+
+import com.sun.jna.Pointer
+import com.sun.jna.Structure
+
+/**
+ * This is a mapping for the `glean_core::byte_buffer::ByteBuffer` struct.
+ * It's a copy of the `support.native.RustBuffer` class in application-services,
+ * with the small changes that the length field is a (32-bit) Int instead.
```

Comment:
@badboy can you explain why this small change is even needed?

Reply:
It's in the commit message!

Reply:
I know :) I was suggesting to add it here as a comment as well, so that we don't need to go through the history to dig up the commit message if we ever stumble upon this :) But if you feel like this is overthinking, I'm fine !